adapter/internal/api/models/error.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by go-swagger; DO NOT EDIT.
 
-// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
As we initially added it in 2020, do we need to change the year?

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1826/commits/3aa9b7de891d72fbd25fdfb61f7dac23d0f11d99